src/templates/core/proposals/add_editors.html

```diff
@@ -44,17 +45,25 @@ <h3>Book Proposal :  {{ proposal.title }}{% if proposal.subtitle%} : {{ proposal
 <div class="container" style="width:100%;left:0%;right:20%;position:relative;">
   <hr>
   {% for field in relationships %}
-
-  <div class="{{ field.width }}">
-  <div class="row">
-    <div class="panel panel-default ">
-      <div class="panel-heading"><span class="panel-heading"><b>{{ field.element.name }} </b></span></div>
-      <div class="panel-body">{% for k,v in data.items %}{% if k == field.element.name %}
-        {% ifequal field.element.field_type 'upload' %}<a href="{% url 'serve_proposal_file_id' proposal.id  v.0 %}">Download</a>{% else %}{{ v.0|safe }}{% endifequal %}{% endif %}{% endfor %}</div>
-    </div>
-  </div>
-  </div>
-{% endfor %}
+      <div class="{{ field.width }}">
+          <div class="row">
+            <div class="panel panel-default ">
+              <div class="panel-heading"><span class="panel-heading"><b>{{ field.element.name }} </b></span></div>
+                <div class="panel-body">
+                  {% for k,v in data.items %}
+                      {% if k == field.element.name|ascii_encode %}
```

Comment:
Exhibit B) And ```ascii_encode``` is used here

Reply:
I see what you mean, but having said that, don't you think it's dangerous to remove all unicode for this purpose? I would suggest it might be advisable to convert this characters to something like an html entity anyway, just in case you get a label which has only unicode, or differing unicode with ascii characters which, if you remove unicode, will end up being exactly the same as another?